indra/llcharacter/llik.h

```diff
@@ -384,15 +395,16 @@ class Joint
     using ptr_t = std::shared_ptr<Joint>;
     using child_vec_t = std::vector<ptr_t>;
 
-    Joint(S16 id, const LLVector3& local_pos, const LLVector3& bone);
+    Joint(LLJoint* info);
+    void resetFromInfo();
```

Comment:
I was loath to add this **LLJoint** dependency on **LLIK::Joint** class however I finally decided it was the way to go, rather than duplicate the data (e.g. `mDefaultLocalPos `and `mDefaultLocalScale`) in **LLIK::Joint**.  I felt the code was a bit more readable this way: the source of the default values is more explicit (and as a bonus we can reduce wasted memory).